src/components/ContentPeek/ContentPeek.tsx

```diff
@@ -28,6 +29,14 @@ export interface ContentPeekProps extends React.HTMLAttributes<HTMLDivElement> {
    * Maximum height of the content when collapsed
    */
   maxHeight?: number;
+  /**
+   * Used to set a minimum height for the content before enabling the Peek functionality. Defaults to the same value as maxHeight if not provided.
+   */
+  minHeightThreshold?: number;
```

Comment:
in what case is this and max height different?

Reply:
The requirements in the ticket are for the Peek to only occur if the content height is > 38rem, but the Peek itself needs to be 30rem. This is to prevent the situation where the content is only a little bit bigger than the size of the Peek, and clicking the "Read More" button would only reveal a few pixels worth of content